dvclive/live.py

```diff
@@ -22,10 +23,16 @@
 class Live:
     DEFAULT_DIR = "dvclive"
 
-    def __init__(self, path: Optional[str] = None, resume: bool = False):
+    def __init__(
+        self,
+        path: Optional[str] = None,
+        resume: bool = False,
+        report: Optional[str] = None,
```

Comment:
Should we assert and raise an error if `report` is something other than `html`?

Reply:
> Should html be the default, at least for now? It matches current behavior when run with dvc.

It already matches the behavior when run with DVC, (at least while the `live` section is not deprecated) with:

https://github.com/iterative/dvclive/blob/90c46ea602a2bd04052b334fab57e6ba04940c36/dvclive/live.py#L84-L85

Reply:
> It already matches the behavior when run with DVC, (at least while the `live` section is not deprecated) with:
> 
> [](https://github.com/iterative/dvclive/blob/90c46ea602a2bd04052b334fab57e6ba04940c36/dvclive/live.py#L84-L85)

True. Does it cause an issue to default to `report = "html"`? Any other reason you wouldn't want it to be the default?

AFAIK the only reason we didn't always generate it before was because we couldn't without dvc. We used to have `html = True` by default.

Reply:
Made `html` default for all cases